tasks/connectors/qualys_was/README.md

```diff
@@ -1,41 +1,41 @@
-# Aqua Toolkit Task to Kenna.VM
+# QualysWas Toolkit Task to Kenna.VM
 
-## This Task will use the Aqua API to
+## This Task will use the QualysWas API to
 
-- Get a list of vulnerabilities currently present in the user's Aqua account
-- Get a list of Workloads in the user's Aqua account associated with each Vulnerability
+- Get a list of webApp currently present in the user's QualysWas account
+- Get a list of Findings in the user's QualysWas account associated with each WebApp
 - Output a json file in the Kenna Data Importer (KDI) format.
 - Post the file to Kenna if API Key and Connector ID are provided
 
 ## Things you will need
 
-- Aqua Username (Required)
-- Aqua Password (Required)
-- Aqua Console Address (Required)
+- QualysWas Username (Required)
+- QualysWas Password (Required)
+- QualysWas Console Address (Required)
```

Comment:
these should match the parameter names in the code. Port is missing too. Also, what is "container data"? Based on the description it is a list of severities? that doesn't seem right either. is there a numerical filter allowed since Qualys scores 1-5?

Reply:
i thik we should change this to. `qualys_was_domain`  as this will be used as prefix/ domain of the api which is `qualysapi.qg3.apps.qualys.com` or 

we can remove  this and hardcode the api urls' as it is now  https://github.com/KennaSecurity/toolkit/blob/a57d942d04c38bcf01d6fbc8d828e8106202eed8/tasks/connectors/qualys_was/lib/qualys_was_helper.rb#L17 


i prefer making it dynamic, what do you suggest

Reply:
yes that has to be dynamic because customers could be in any of 8+ domains

Reply:
and i like the name change

Reply:
name change ?